x/src/cargo.rs

```diff
@@ -128,9 +141,10 @@ impl Cargo {
 }
 
 #[derive(Debug, Default)]
-pub struct CargoArgs {
+pub struct CargoArgs<'a> {
     pub all_features: bool,
     pub all_targets: bool,
+    pub extra_env: Vec<&'a str>,
```

Comment:
This addition looks unneeded, lets drop it

Reply:
done